backend/controllers/alert.js

```diff
@@ -100,6 +100,14 @@ const alertFactory = (oauthApi, prisonApi, referenceCodesService) => {
         oauthApi.userRoles(res.locals),
       ])
 
+      if (alert && alert.expired) {
```

Comment:
Can do `alert?.expired`

Reply:
Yeah sounds sensible. I only moved that code, but i will fix